torch/csrc/jit/mobile/backport_manager.cpp

```diff
@@ -212,6 +213,35 @@ bool backport_v5_to_v4(
   return true;
 }
 
+bool backport_v6_to_v5(
+    std::stringstream& input_model_stream,
+    std::stringstream& output_model_stream) {
+  std::shared_ptr<IStreamAdapter> rai =
+      std::make_shared<IStreamAdapter>(&input_model_stream);
+  auto reader = std::make_shared<PyTorchStreamReader>(rai);
+
+  // If there are debug info files in the original model file, it should also
+  // show up in the backported model
+  bool hasBytecodeDebug = reader->hasRecord("mobile_debug_handles.pkl");
+  // extra_files are kept
+  ExtraFilesMap extra_files;
+  // Loading the TS module is required for this backport, because bytecode needs
+  // to be re-emitted (refer to the comments below)
+  Module torch_script = torch::jit::load(rai, c10::nullopt, extra_files);
+
+  // The RAII guard to change the flag, emitBytecodeDefaultInputs, to true, so
+  // that TS stores the default argument values in the constant table, and emits
+  // the instructions (LOADC, for example), to push the values to the stack. It
+  // restores the behavior of V5 and before. For V6, the default arg values are
+  // resolved at runtime init stage for better operator compatibility.
+  BytecodeEmitDefaultInputsGuard argNumGuard(true);
```

Comment:
Nit:

Place `argNumGuard` in a tighter scope to avoid accidental leak of guard effects